hal/linux/package/agent/agent.mk

```diff
@@ -12,7 +12,14 @@ define AGENT_BUILD_CMDS
 endef
 
 define AGENT_INSTALL_TARGET_CMDS
-	cp $(@D)/build/cocos-agent $(TARGET_DIR)/bin
+	$(INSTALL) -D -m 0755 $(@D)/build/cocos-agent $(TARGET_DIR)/bin
+	mkdir -p $(TARGET_DIR)/var/log/cocos
 endef
 
-$(eval $(golang-package))
\ No newline at end of file
+define AGENT_INSTALL_INIT_SYSTEMD
+	$(INSTALL) -D -m 0644 $(@D)/init/systemd/cocos-agent.service $(TARGET_DIR)/usr/lib/systemd/system/cocos-agent.service
+	$(INSTALL) -D -m 0644 $(@D)/init/systemd/eth0.network $(TARGET_DIR)/etc/systemd/network
+	$(INSTALL) -D -m 0644 $(@D)/init/systemd/00-internal0.link $(TARGET_DIR)/etc/systemd/network
```

Comment:
I am not sure that you need to prefix this with `00-`. How is `systemd` establishing the order of service / init script deployment? I think you need to use `After=` or similar directives: https://serverfault.com/questions/482730/systemd-dependencies-and-boot-order, https://www.digitalocean.com/community/tutorials/understanding-systemd-units-and-unit-files